test/users/test_profile_picture.py

```diff
@@ -0,0 +1,27 @@
+from PIL import Image
+from flaskapp import models
+from test.main.base_classes import BaseUser
+from test.main.utils import test_post_request
+
+
+class UserAccountTestCase(BaseUser):
+    def test_set_profile(self):
+        self.login()
+        im1 = Image.open("profile1.png")
+        set_user = dict(username="nr.nutron", email="nutron@gmail.com",
+                         picture=im1, submit="submit")
+        test_post_request(self, "/account", set_user, models.User, 1)
+
+    def test_update_profile(self):
+        self.login()
+        # first set profile
+        im1 = Image.open("profile1.png")
+        set_user = dict(username="nr.nutron", email="nutron@gmail.com",
```

Comment:
It won't work.
See how they are sending image https://stackoverflow.com/questions/29104107/upload-image-using-post-form-data-in-python-requests